src/views/DandisetLandingView/DandisetUnembargo.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <v-card
+    outlined
+    class="mt-4 px-3"
+  >
+    <v-dialog
+      v-if="showWarningDialog"
+      v-model="showWarningDialog"
+      persistent
+      max-width="60vh"
+    >
+      <v-card class="pb-3">
+        <v-card-title class="text-h5 font-weight-light">
+          Release Dandiset
+        </v-card-title>
+        <v-divider class="my-3" />
+        <v-card-text>
+          <span>
+            This action will unembargo this dandiset. Note that this is a
+          </span>
+          <span class="font-weight-bold">
+            permanent
+          </span>
+          <span>
+            action and is not undoable. Once a dandiset has been released,
+            it will be public forever.
+            <br><br>
+            Note: this may take a while if your dandiset is large.
+          </span>
+        </v-card-text>
+        <v-card-text>
+          Would you like to proceed?
```

Comment:
This is a case where we should ask the user to take a deliberate action to indicate they know what they're doing. Typing a random phrase in is a good option (Multinet does this), as would be making the user type in the dandiset ID.